compiler-core/src/javascript/tests/records.rs

```diff
@@ -53,3 +53,18 @@ pub type Person {
 pub fn get_age(person: Person) { person.age }"
     );
 }
+
+#[test]
+fn record_accessor_multiple_variants_optional_types() {
+    // We can access fields on custom types with multiple variants
+    // In positions other than the 1st field
+    assert_js!(
+        "
+pub type Person {
+    Teacher(name: String, age: Option(Int), title: String)
+    Student(name: String, age: Option(Int))
+}
+pub fn get_name(person: Person) { person.name }
+pub fn get_age(person: Person) { person.age }"
+    );
+}
```

Comment:
End of file newline please.

In vscode this is `"files.insertFinalNewline": true`, good to have it globally as text files on *nix system should always have them.

Reply:
Fixed